### ISSUE TITLE
Batch install feature

### ISSUE THREAD

**Issue by vertigo220** [chocolatey/choco#1566]
Chocolately needs the ability to search for and mark packages for installation, then have it go through and install them all at once. This way, they could be downloaded and installed overnight.

**Comment by gep13** [comment]
@vertigo220 I believe that this is covered in this feature request:

https://github.com/chocolatey/chocolatey.org/issues/39

If so, can you close this issue?  Thanks!

**Comment by vertigo220** [comment]
The shopping cart idea mentioned there seems to be essentially the same thing, though that seems to be talking about building a list of programs to install on the website then having it make a script to do it, whereas I'm talking about selecting multiple items (adding them to a cart) in the GUI then having it go through and install them all at once. So I'll make an issue for it in the GUI repo and reference that issue, then close this one. Thanks.

**Comment by ferventcoder** [comment]
@vertigo220 also see https://www.windowsremix.com/

**Comment by gep13** [comment]
> @vertigo220 said...
So I'll make an issue for it in the GUI repo and reference that issue, then close this one. Thanks

Sounds good, thanks!

**Comment by bcurran3** [comment]
@vertigo220 - until this gets baked into Chocolatey, you might want to also look at https://chocolatey.org/packages/choco-chrome-ext which lets you create lists and then install that list.

@gep13 - A shopping cart in Chocolatey GUI sounds like a great idea! @vertigo220 please open that feature request and I'll thumbs up it!

**Comment by gep13** [comment]
@bcurran3 did you have a look at the windowsremix.com site that @ferventcoder linked to above?  I am curious what your thoughts on it are?

**Comment by bcurran3** [comment]
@gep13 - Yeppers. Reminded me instantly of https://ninite.com/ but for legally blind people.

I love the concept used by both sites and think it would be a boon to Chocolatey to adopt some such interface. I love Chocolatey but it's not very "normal people" friendly. Chocolatey needs something like the above sites to move into mass market and not just techies and devs.

I'm ALMOST of the believe that when Chocolatey is installed that ChocolateyGUI should install along with it by default.

**Comment by gep13** [comment]
> @vertigo220 said...
When I first installed Chocolatey I didn't even realize there was a GUI, and I couldn't get the CLI to work (I posted in the chat about it but never got a response

Apologies for this.  Normally there is always someone around in there to help.

**Comment by vertigo220** [comment]
No worries. I wasn't in a huge rush and realize you guys have a lot on your plate, so wasn't too bothered by it, especially since I figured out there was a GUI and it did start working. Though I will say there didn't seem to be a mechanism to ensure a post is seen (i.e. after I posted my issue, others posted theirs, and by the time a dev/moderator came around, I'm not sure they even saw mine), and I had to dig through a bunch of messages to see if I had been responded to.
